haiku-apps/wolle/wolle-0.1~git.recipe

```diff
@@ -0,0 +1,45 @@
+srcGitRev="be9165f52bee14d3cd1d19600ce6cfa44f217df4"
+SUMMARY="A screensaver displaying  woollen string"
+DESCRIPTION="Wolle is a screensaver which draws a woollen string over 
+your desktop"
+HOMEPAGE="https://github.com/HaikuArchives/Wolle"
+COPYRIGHT="2001 - 2008 , Werner Freytag.
+2009, Haiku.
+"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/localanu/Wolle/archive/$srcGitRev.tar.gz" #temporarily,waiting my pr merged
+CHECKSUM_SHA256="1eed342ebfcd0615b57b50150468726e2c62898c47d8ff1ec91d3ba23884d6c5"
+SOURCE_FLIENAME="Wolle-$srcGitRev.tar.gz"
+SOURCE_DIR="Wolle-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	wolle = $portVersion
+	addon:wolle = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:make
+	makefile_engine
+	"
+
+BUILD()
+{
+	make
+}
+
+INSTALL()
+{
+	mkdir -p $addOnsDir/Screen\ Savers
```

Comment:
I would get rid of those `\` between "Screen" and "Savers" if I were you.
`install -d '$addOnsDir/Screen Savers'` and `install -t '$addOnsDir/Screen Savers' objects/Wolle` look so much better :)

Reply:
Well, I think the backslash is OK.
If, however, there were more than 2 backslashes, then I would have recommended dropping them and, instead, using **double** quotes.
(**Simple** quotes would not work, because they prevent the evaluation of any environment variable.)